src/global/munge_engines.R

```diff
@@ -190,10 +190,12 @@ munge_combined <- function(network, domain, site_code, prodname_ms, tracker,
 
             if(is_spatial){
 
-                out <- data.table::rbindlist(list(out, out_comp),
-                                             use.names = TRUE,
-                                             fill = TRUE) %>%
-                    as_tibble()
+                # out <- data.table::rbindlist(list(out, out_comp),
+                    #                          use.names = TRUE,
+                    #                          fill = TRUE) %>%
+                    # as_tibble()
+                
+                out <- rbind(out, out_comp)
```

Comment:
@spencerrhea this will fail if the column names aren't identical. We might need to call rbindlist/rbind/bind_rows conditionally

Reply:
@vlahm I looked though the munge and processing_kernels and it looks like a spatial out_comp will always (or should always) only have the columns site_code and geometry. Does that sound right?